File: javascript/SomaDeConjunto.js
var input = require('fs').readFileSync('javascript/stdin', 'utf8');
//ESTUDAR
var sep = [' ', '\n'];
var line = input.split(new RegExp(sep.join('|'), 'g'));

let n, num, soma = 0, i = 0;

console.log("Quantos numeros a serem somados:");
n = parseInt(line.shift());
console.log(n);

while (i < n) {
    i = i + 1;
    console.log("Entre com o " + i + " numero");
    num = parseInt(line.shift());
    console.log(num);
    soma = soma + num;
}
console.log("A soma da: " + soma);
    

File: java/src/SomaDeConjunto.java
import java.util.Scanner;

public class SomaDeConjunto {

    public void somaDeConjunto() {
        Scanner input = new Scanner(System.in);
        int n, num, soma = 0, i = 0;
        System.out.println("Quantos numeros a serem somados:");
        n = input.nextInt();
        while (i < n) {
            i = i + 1;
            System.out.println("Entre com o " + i + " numero");
            num = input.nextInt();
            soma = soma + num;
        }
        System.out.println("A soma da: " + soma);
    }
}
